source/com.microsoft.tfs.core/rest_core/com/microsoft/vss/client/core/model/VssException.java

```diff
@@ -3,7 +3,7 @@
 
 package com.microsoft.vss.client.core.model;
 
-import com.microsoft.vss.client.core.utils.StringUtil;
```

Comment:
Are we sure that this change is correct? Is tfs.util.StringUtil.* a perfect match for vss.client.core.utils.StringUtil.*?